### ISSUE TITLE
Add test data for pa11y accessibility scans on domain and domain application pages

### ISSUE THREAD

**Issue by neilmb** [cisagov/manage.get.gov#714]
### Ticket Description

Our automated accessibility scanner pa11y visits URLs that are specified in a `.pa11yci` file. We use a special workaround for logged-in requests so that pa11y can visit those pages. However, for pages backed by the database like `/domain/<id>` and `/application/<id>` we don't have pa11y scan those pages because there is no data at those URLs that is the scanner can access. We should fix this so that pa11y can test the domain and domain application pages.

One way to do this is to load a specific set of fixture data before we ran the pa11y scan. A minimal set would be a single domain application and a domain owned by the default user (username "Testy" see `registrar/tests/common.py:MockUserLogin.__call__`).  There is a helper method that we use in our tests called `_completed_application` that could be used for making a domain application. One of the difficult aspects of this process is that we need these items to be accessible at stable database IDs so that we can write those URLs in the pa11y configuration file.

After we have a way to load the test data at a specific ID (say 1), then we should add these URLs to `.pa11yci`

```
"http://localhost:8080/application/1",
"http://localhost:8080/application/1/withdraw",
"http://localhost:8080/domain/1",
"http://localhost:8080/domain/1/nameservers",
"http://localhost:8080/domain/1/org-name-address",
"http://localhost:8080/domain/1/authorizing-official",
"http://localhost:8080/domain/1/your-contact-information",
"http://localhost:8080/domain/1/security-email",
"http://localhost:8080/domain/1/users",
"http://localhost:8080/domain/1/users/add",
```

### Issue Type

development

### Prioritization Comments
**WCAG Conformance Level:** 
**WCAG Standard that applies:** 
**Critical Failure if Not Met?:**  
**Prioritization:** 
**Reason:**

**Comment by PaulKuykendall** [comment]
@neilmb - Can you give this a fresh look and weigh in on whether this is critical for Jan launch?? Thanks!

**Comment by katypies** [comment]
Flagged this to the accessibility pod to review in case it's something they want to recommend we do in the future, but closing for now as moot. The pod is looking at a broader approach to improving our accessibility tooling/approach alongside their compliance work.
